src/message-processor.js

```diff
@@ -162,6 +162,37 @@ var executeStepPositionsRequest = function (request) {
   this._emit(response);
 };
 
+var getImplementationFiles = function(request) {
+  var response = factory.createImplementationFileListResponse(this.options.message, request.messageId);
+  var files = impl_loader.getImplFileList(GAUGE_PROJECT_ROOT);
+  response.implementationFileListResponse.implementationFilePaths = files;
+  this._emit(response);
+};
+
+var putStubImplementationCode = function(request) {
+  var response = factory.createFileChangesResponse(this.options.message, request.messageId);
+  var filePath = request.stubImplementationCodeRequest.implementationFilePath;
+  response.fileChanges.fileName = filePath;
+  var stepTexts = request.stubImplementationCodeRequest.steps;
+  var codes = [];
+  stepTexts.map(step => {
+    var argCount = 0;
```

Comment:
we are already generation the stub while validating steps. We should be reusing the same logic.